impl/src/main/java/org/ehcache/impl/internal/store/DefaultStoreProvider.java

```diff
@@ -46,10 +49,22 @@
 public class DefaultStoreProvider implements Store.Provider {
```

Comment:
Discussed with @chrisdennis -- the functionality currently performed by `DefaultStoreProvider` can be subsumed into `CacheStore.Provider` along with changes to `CacheStore.Provider` to have it be specific about what resource/config combinations it supports.  Likely needs additional discussion but this work will be in another PR ... possibly combined with the `XAStore.Provider` changes.